src/main/java/teammates/common/datatransfer/questions/FeedbackConstantSumQuestionDetails.java

```diff
@@ -92,9 +89,15 @@ public boolean extractQuestionDetails(
 
         int points = 0;
         if (pointsPerOption) {
-            points = distributeToRecipients ? Integer.parseInt(pointsForEachRecipientString)
-                                            : Integer.parseInt(pointsForEachOptionString);
+            if (distributeToRecipients) {
+                Assumption.assertNotNull("Null points for each recipient", pointsForEachRecipientString);
+                points = Integer.parseInt(pointsForEachRecipientString);
+            } else {
+                Assumption.assertNotNull("Null points for each option", pointsForEachOptionString);
+                points = Integer.parseInt(pointsForEachOptionString);
+            }
```

Comment:
Something like this is more concise?
```java
String someSuitalbeName = distributeToRecipients? pointsForEachRecipientString : pointsForEachOptionString;
Assumption.assertNotNull("Null points for each recipient", someSuitalbeName);
points = Integer.parseInt(someSuitalbeName);
```

Reply:
@damithc Sir, how about the name `pointsForEachOption` for the variable? As `pointsForEachRecipientString` and `pointsForEachOptionString` are both at its core, points for each option only?

Reply:
We need a name that represents both `pointsForEachRecipientString` and `pointsForEachOptionString` without implying either one.
just `pointsString`?

Reply:
We are already using `pointsString` for `Total points to distribute` field.

Reply:
@damithc I guess we can rename the variable name for `Total points to distribute` from `pointsString` to `pointsTotalString`, and use `pointsString` for the above case?

Reply:
Sure. although `totalPointsString` sounds easier read.